cs3api4lab/api/share_api_facade.py

```diff
@@ -39,7 +39,10 @@ def __init__(self, log):
     def create(self, endpoint, file_path, opaque_id, idp, role=Role.EDITOR, grantee_type=Grantee.USER, reshare=True):
         """Creates a share or creates an OCM share if the user is not found in local domain"""
         if self._is_ocm_user(opaque_id, idp):
-            return self.ocm_share_api.create(opaque_id, idp, idp, endpoint, file_path, grantee_type, role, reshare)
+            if self.config['enable_ocm']:
+                return self.ocm_share_api.create(opaque_id, idp, idp, endpoint, file_path, grantee_type, role, reshare)
+            else:
```

Comment:
We should fail as this request should not reach us. `501`? 
We should do this for all other cases where it applies such an error.